ui/app/templates/components/secret-list/ssh-role-item.hbs

```diff
@@ -43,9 +43,7 @@
                 {{#if (eq @item.keyType "otp")}}
                   {{#if @item.generatePath.isPending}}
                     <li class="action">
-                      <button disabled type="button" class="link button is-loading is-transparent">
-                        loading
-                      </button>
+                      <Hds::Button disabled @color="tertiary" @icon="loading" @text="loading" @isIconOnly={{true}} />
```

Comment:
<img width="292" alt="Screenshot 2023-10-11 at 7 01 38 PM" src="https://github.com/hashicorp/vault/assets/68122737/61f8076e-fd0d-45f6-bdae-0859af2de373">

Reply:
These were previously centered - but eventually all of our popup menus will be replaced by `<Hds::Dropdown>` so it seemed moot to wrestle CSS while this is temporary (these loading icons also seldom show and are more of a fail safe)